src/views/AccountSettings.js

```diff
@@ -13,21 +13,21 @@ import PrivacySettings from "../user/components/PrivacySettings";
 import SecuritySettings from "../user/components/SecuritySettings";
 import Spinner from "../app/components/Spinner";
 import Button from "../app/components/Button";
-import { checkJwt } from "../auth/auth-helpers";
+import { checkAuthExpiry } from "../auth/auth-helpers";
 
 function UserSettings({ history }) {
   const profileDispatch = useProfileDispatch();
   const { profileData } = useProfileState();
 
-  const { jwt, userAddress } = useAuthState();
+  const { userAddress, authExpiry } = useAuthState();
   // Profile settings
   const [newUsername, setNewUsername] = useState("");
   const [newEmail, setNewEmail] = useState("");
   const [newLocation, setNewLocation] = useState("");
   const [newBio, setNewBio] = useState("");
 
   const [isLoading, setIsLoading] = useState(false);
-  const [isError, setIsError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(``);
```

Comment:
Add two types of error hooks, one for fetching data and one for editing profile data